reference/Blaseball-API.v2.yaml

```diff
@@ -0,0 +1,484 @@
+openapi: 3.1.0
+x-stoplight:
+  id: a3ns7gfbb7au6
+info:
+  title: Scraped.v0
```

Comment:
I'd title this something like "Blaseball API 2" instead.

Reply:
Done.